code/modules/projectiles/guns/projectile/revolver.dm

```diff
@@ -52,6 +52,39 @@
 	ammo_type = /obj/item/ammo_casing/a454
 	magazine_type = /obj/item/ammo_magazine/a454
 
+/obj/item/gun/projectile/revolver/mateba/captain
+	name = "\improper SCC command autorevolver"
+	desc = "A ludicrously powerful .454 autorevolver with equally ludicrous recoil which is issued by the SCC to the administrators of critical facilities and vessels. While revolvers may be a thing of the past, the stopping power displayed by this weapon is second to none."
```

Comment:
I'm not an english expect by any means, but the "which is issued" part feels like it should be preceded by a comma, or the "which" be a "that"